### ISSUE TITLE
CUDA error 222 at 

### ISSUE THREAD

**Issue by qustions** [zhouyuchong/face-recognition-deepstream#7]
@zhouyuchong  when trying to create engine file of yolo getting this error in deepstream 6.1 but the same was working in deepstream 6.0 any idea 
```
CUDA error 222 at /opt/nvidia/deepstream/deepstream-6.1/sources/deepstream_python_apps/apps/face-recognition-deepstream/convetor/tensorrtx/yolov5/yololayer.cu:37yolov5: /opt/nvidia/deepstream/deepstream-6.1/sources/deepstream_python_apps/apps/face-recognition-deepstream/convetor/tensorrtx/yolov5/yololayer.cu:37: nvinfer1::YoloLayerPlugin::YoloLayerPlugin(int, int, int, int, const std::vector<Yolo::YoloKernel>&): Assertion `0' failed.
Aborted (core dumped)

```

**Comment by qustions** [comment]
@zhouyuchong  I am using default deepstream:6.1-triton docker image. How are you running ?

**Comment by zhouyuchong** [comment]
@qustions I have no idea. Seems there is something wrong with your pt file or cuda? check [this](https://github.com/wang-xinyu/tensorrtx/issues/440#issuecomment-1178483486)

**Comment by zhouyuchong** [comment]
my host:
+ ubuntu 20
+ nvidia driver 510
+ no cuda/trt/opencv/torch
container 1:
+ deepstream 6.1:dev
where I run my codes.
container 2:
+ deepstream 6.1:dev
+ opencv 4.2
+ cuda 11.6
+ torch
I trans Yolo and other model to onnx/trt in container 2 without error. However, I remembered that while I tried to trans arcface-r100, something wrong and it was very similar to your error. But I didn't solve it, I just trans arcface-mobilefacenet and this time everything is all right.
